core/controllers/editor.py

```diff
@@ -953,11 +956,15 @@ def put(self, exploration_id):
         except utils.ValidationError as e:
             # We leave any pre-existing draft changes in the datastore.
             raise self.InvalidInputException(e)
-
+        exp_user_data = user_models.ExplorationUserDataModel.get(
+            self.user_id, exploration_id)
+        draft_change_list_id = (exp_user_data.draft_change_list_id
```

Comment:
Ah no it shouldn't be possible. It should be fine to remove the check it that case. Done.